http/headers/www-authenticate.json

```diff
@@ -48,6 +48,346 @@
+                },
+                "chrome_android": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": true
+                },
+                "firefox_android": {
+                  "version_added": true
+                },
+                "ie": {
+                  "version_added": null
+                },
+                "opera": {
+                  "version_added": true
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": null
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "samsunginternet_android": {
+                  "version_added": true
+                },
+                "webview_android": {
+                  "version_added": true
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          }
+        },
+        "Negotiate": {
+          "__compat": {
+            "description": "<code>Negotiate</code> (Kerberos) authentication",
+            "spec_url": "https://www.rfc-editor.org/rfc/rfc4120#top",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              },
+              "samsunginternet_android": {
+                "version_added": true
+              },
+              "webview_android": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "NTLM": {
+          "__compat": {
+            "description": "<code>NTLM</code> authentication",
+            "spec_url": "https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-ntht/#published-version",
```

Comment:
OK, it looks like we need to omit this here. See https://github.com/mdn/browser-compat-data/pull/13383#issuecomment-971825712.

We can still link to this spec from the MDN page if you like (e.g., from the specification section or see also section), but it's probably best to highlight that it's not a standard, exactly.

```suggestion
```